app/javascript/components/applications/ApplicationDetails.js

```diff
@@ -1,24 +1,17 @@
 import React from 'react'
+import _ from 'lodash'
```

Comment:
Not sure how we are handling this elsewhere, but if you import only what you need (i.e. `import { map } from 'lodash'`) you can save quite a bit on final bundle size.